src/TLuaInterpreter.cpp

```diff
@@ -6576,6 +6576,14 @@ int TLuaInterpreter::exists(lua_State* L)
         cnt += host.getAliasUnit()->mLookupTable.count(name);
     } else if (type == "keybind") {
         cnt += host.getKeyUnit()->mLookupTable.count(name);
+    } else if (type == "script") {
```

Comment:
As mentioned above - don't redo things in multiple PRs - this is the subject of #2195 - so don't put it in here.